disk/test/com/treode/disk/edit/LogReaderSpec.scala

```diff
@@ -0,0 +1,79 @@
+package com.treode.disk
+
+import java.nio.file.{Paths, StandardOpenOption}
+import com.treode.async.{Async, Scheduler}
+import com.treode.async.io.File
+import com.treode.async.io.stubs.StubFile
+import com.treode.async.stubs.StubScheduler
+import scala.util.{Failure, Success}
+import com.treode.buffer.PagedBuffer
+import com.treode.async.Async
+import com.treode.async.Async.async
+import org.scalatest._
+import com.treode.async.stubs.implicits._
+import com.treode.async.Globals
+
+class LogReader (file: StubFile, geom: DriveGeometry) {
+   
+   var buf = PagedBuffer(12)
+   var pos = 0
+   var end = 1
+   
+   def read(): Async[Seq[String]] = {
```

Comment:
This requires the user to call read repeatedly, and each time it yields a sequence of length 1. We want to allow the user to call read once, and this returns a sequence of all the strings written.